packages/@sanity/components/src/buttons/story.js

```diff
@@ -144,6 +144,7 @@ storiesOf('Buttons', module)
               onClick={action('clicked')}
               disabled={boolean('disabled', false, 'props')}
               inverted={boolean('inverted', false, 'props')}
+              showArrow={boolean('showArrow', true, 'props')}
```

Comment:
Is this added or removed? Can't see any changes related to this?

Reply:
I think it is ok to add knobs to storybook continuously when needed.